src/foam/dao/QueryCachingDAO.js

```diff
@@ -60,11 +60,14 @@ foam.CLASS({
 
 
           // Return data from cache
+          // only add elements from cache that werent fetched directly from the dao
```

Comment:
Should this be part of this PR?

Reply:
Yes, without it views using ArraySinks have duplicate table entries